### PR TITLE
added_case_to_main_page

### PR DESCRIPTION
**Pull request by Mouroof**

### CODE REVIEW

File: web/templates/home/index.html [hunk 1]
```diff
@@ -2,4 +2,18 @@
 
 <h1>Welcome to iTrust: Zakat management application</h1>
 <h3>Under Construction ...</h3>
-{% endblock %}
+
+{% for case in cases %}
+<div class="card">
+    <div class="card-body">
```
Comment (ik-nitk) [inline web/templates/home/index.html:5]: Can we have different sections here ?  Open /  Pending Verification /  Pending Voting etc ?

File: web/home/views.py [hunk 2]
```diff
@@ -8,4 +8,12 @@
 
 @blueprint.route("/")
 def home_page():
-    return render_template("home/index.html")
\ No newline at end of file
+    api = current_app.config.get('api')
+    session = current_app.config.get('session')
+    url = api.case_list_from_case_state('PUBLISHED')
+    print(url)
```
Comment (ik-nitk) [inline web/home/views.py]: Remove prints in code

**Review by ik-nitk — COMMENTED** [review]
Can you please address the comments.

File: web/templates/home/index.html (hunk 1 above)
Comment (Mouroof) [reply web/templates/home/index.html:5]: To have different sections we need to list all the cases irrespective of case state right.

Comment (ik-nitk) [reply web/templates/home/index.html:5]: put different state in different cards. and keep in same page.  Or you can keep in different tabs in same card .

Comment (Mouroof) [reply web/templates/home/index.html:5]: Ok

**Pull request merged**